web/src/app/dialogs/new-inspection/components/service/parameter-store.ts

```diff
@@ -107,4 +115,24 @@ export class DefaultParameterStore implements ParameterStore {
   public destroy(): void {
     this.destroyed.next(void 0);
   }
+
+  private deepEqual(
```

Comment:
It's technically not deep equal because it does not compare value deeply. However, it should work as long it's assured that values in object never be object or array etc. We might want to add comment about this limitation if so.

Reply:
Renamed it to `haveEqualKeyValues`